device/CMakeLists.txt

```diff
@@ -24,6 +28,8 @@ add_library(device SHARED ${UMD_DEVICE_SRCS})
 add_library(${PROJECT_NAME}::device ALIAS device)
 add_library(${PROJECT_NAME}_device ALIAS device) # For legacy I guess
 
+add_dependencies(device flatbuffer_generated_header)
```

Comment:
Not sure if this is something I should be doing? But I'm also unsure how to get this to work otherwise..  :(

Reply:
Try this:

```
target_sources(device
  PRIVATE
    path/to/generated/header.h
)
```
If the header is part of the API, then use `PUBLIC` instead.
With CMake this old you may still need to keep the `add_custom_target`, or we may be able to drop it.  I'm not certain.

Reply:
Works tyty 🙏